src/eos-google-chrome-installer

```diff
@@ -47,7 +47,7 @@ def exit_with_error(*args):
 
 class GoogleChromeInstaller:
 
-    GSETTINGS_SCHEMA = "com.endlessm.GoogleChromeInitialSetup"
+    GSETTINGS_SCHEMA = "com.endlessm.GoogleChromeHelper"
```

Comment:
After our conversation during the standup @rshuler made me see that this change is actually not enough to keep this working for users that started by flashing a 3.1.0 image, even if in some cases (e.g. single-user installations) won't make a difference, and the reason I see is that, this change won't work as expected in at least the following scenario:

1. User flashes a 3.1.0 image with this setting's default set to true **in the image builder** (e.g. eosfree), and then get Chrome automatically installed and thus the old `enabled` GSetting set to `false` after the process, and the stamp file created.

2. User upgrades to 3.1.1 and get the new GSetting schema installed + the outdated helper script that looks into the new key too. Unfortunately, not only the schema and key changed for this user now, but **also the default value**: as this upgrade came via OSTree, the default for that GSetting will be `false` as there was no chance for the image-builder here to override it to `true`, which will continue to be the installed default for the old -and now deprecated- key.

Thus, even if I agree with you that this will probably mean not much of a difference for the typical user, that will still have the stamp file "protecting" his session for ever spawning this again, it's true there's a difference as the effective default for this setting will be lost during an upgrade.

Thus, I think this should be a matter of simply adding some code to check the old key too and enable the feature if either that one or the new one is effectively `true` maybe?. Note that, as per my other request in this PR, the GSetting should from now on never be set to false if it was previously set to `true`, so I think that should work